custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -186,6 +176,41 @@ def allowed_to_forward(self, test_case):
             return False
 
 
+class NikshayRegisterPrivatePatientRepeater(BaseNikshayRepeater):
+    operation = 'InsertHFIDPatient_UATBC'
+    white_listed_case_types = ['episode']
```

Comment:
I don't think this will work because when you save the view, it will overwrite this property.